app/helpers/permission-to-remove-contributor.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+
+/**
+* permissionToRemoveContributor helper.  Checks to see if user has proper permissions
+* to remove contributor.  The user either must be an admin or trying to remove herself.
+* The project cannot be a registration.
+*/
+export function permissionToRemoveContributor(params/*, hash*/) {
+    var contributor = params[0];
+    var currentUser = params[1];
+    var stillAdmin = params[2];
+    var node = params[3];
+    if (currentUser) {
+        var currentUserId = currentUser.get('currentUserId') || currentUser.get('id');
+        var removeSelf = contributor.id.split('-')[1] === currentUserId;
```

Comment:
See note elsewhere about helpers. Just the fact that you're doing this so many times in your PR is a good sign that it makes sense to centralize the common logic. :)

(One rough rule of thumb: if you see it twice, consider whether a helper would be useful; three or more times, think about refactoring)